src/Stepper/Step.tsx

```diff
@@ -1,4 +1,4 @@
-import React, { FunctionComponent, ReactNode, memo, KeyboardEvent } from 'react'
+import React, {FunctionComponent, ReactNode, memo, KeyboardEvent} from 'react'
```

Comment:
These formatting changes seem unnecessary... I think the convention for this project would be to keep these spaces. We might need to align our VS Code formatter settings for this project? Yet another tech debt item on my to do list. 

Sorry to be pedantic, but can we revert these formatting changes for now?